client/payment-details/summary/index.tsx

```diff
@@ -367,18 +413,76 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 						<CardFooter className="payment-details-capture-notice">
 							<div className="payment-details-capture-notice__section">
 								<div className="payment-details-capture-notice__text">
-									{ `${ __(
-										'You need to capture this charge before',
-										'woocommerce-payments'
-									) } ` }
-									<b>
-										{ dateI18n(
+									{ createInterpolateElement(
+										__(
+											'You need to <a>capture</a> this charge in ',
+											'woocommerce-payments'
+										),
+										{
+											a: (
+												// eslint-disable-next-line jsx-a11y/anchor-has-content, react/jsx-no-target-blank
+												<a
+													href="https://woocommerce.com/document/woocommerce-payments/settings-guide/authorize-and-capture/#capturing-authorized-orders"
+													target="_blank"
+													rel="noreferer"
+												/>
+											),
+										}
+									) }
+									<abbr
+										title={ dateI18n(
 											'M j, Y / g:iA',
-											moment
-												.utc( authorization.created )
-												.add( 7, 'days' )
+											new Date(
+												Date.UTC(
+													new Date(
+														authorization.created
+													).getFullYear(),
+													new Date(
+														authorization.created
+													).getMonth(),
+													new Date(
+														authorization.created
+													).getDate() + 7,
+													new Date(
+														authorization.created
+													).getHours(),
+													new Date(
+														authorization.created
+													).getMinutes(),
+													new Date(
+														authorization.created
+													).getSeconds()
+												)
+											)
 										) }
-									</b>
+									>
+										<b>
+											{ relativeTimeRemaining(
+												new Date(
+													Date.UTC(
+														new Date(
+															authorization.created
+														).getFullYear(),
+														new Date(
+															authorization.created
+														).getMonth(),
+														new Date(
+															authorization.created
+														).getDate() + 7,
```

Comment:
This does not seem to be correct. If I understand correctly, we are creating a new Date object, getting the date and then adding `7.` But if the date is 31 then we would end passing 38 to the date argument of Date.UTC, which I think is incorrect (it should be an integer between 1 and 31). I'd go with a different approach: first parse the `authorization.created` to have an integer timestamp, so you could add the seven days and calculate the time remaining. I am thinking that moment could be handy in this case 🥲 

Also, I think we should try to avoid the repetition of Date here. Maybe this can be handled inside the `relativeTimeRemaining` function?

Reply:
Ah.. I missed this simple detail ( about 7 being added to the date without checking the possibility of largest date of the month getting exceeded ). I have implemented the whole thing with Moment, that made it simpler too. I would have loved to do this with vanilla JS, but looks like moment results in crisp and readable code.